Demo/ImagePickerDemo/ImagePickerDemo/Info.plist

```diff
@@ -24,6 +24,8 @@
 	<true/>
 	<key>UILaunchStoryboardName</key>
 	<string>LaunchScreen</string>
+	<key>UIMainStoryboardFile</key>
```

Comment:
@vadymmarkov I think in this demo, storyboard allows us to focus on the ImagePicker code, without having to worry about Auto Layout stuff